test/SmokeTests/OOProcSmokeTests/DotNetIsolated/host.json

```diff
@@ -7,5 +7,14 @@
         "excludedTypes": "Request"
       }
     }
-  }
+  },
+  "extensions": {
+    "durableTask": {
+      "storageProvider": {
+        "maxQueuePollingInterval": "00:00:01",
+        "controlQueueVisibilityTimeout": "00:01:00"
```

Comment:
this just helps make sure we retry orchestrator replays fast enough after a platform-level error.